### ISSUE TITLE
enforce_authorization option in config.yaml

### ISSUE THREAD

**Issue by mgaruccio** [vmware/container-service-extension#241]
Attempted to run a template update using the 1.2.7 release and received an error regarding a missing node within the service portion of the config file.  After running a `cse sample` it appears a new option called `enforce_authorization` has been added to the file with a default value of false but there is no mention of this setting within the documentation.  What is the function of the new option and should it typically be set to `true` or `false`?

**Comment by andrew-ni** [comment]
`enforce_authorization` was added in CSE 1.2.6 (PR #210)

Seems like we did not properly document this addition as well as our current process for updating the config file when CSE has a new version. A PR will be out soon to address this.

Our current process for updating the config file when CSE has a new version is to run `cse sample --output myconfig.yaml` and manually update this new up-to-date config file. We're considering plans in the future to better streamline this config file creation process to make it more user-friendly.

`enforce_authorization` is a key that goes under `service` section of the config file. It's value can be `true` or `false`. This key toggles Role-based Access Control for CSE in vCD, where if authorization is enabled, a user can only deploy clusters if their role has the correct CSE deploy right. (Check out https://vmware.github.io/container-service-extension/RBAC.html for more info on this)
